examples/ws2812/digispark/main.go

```diff
@@ -0,0 +1,44 @@
+// Connects to an WS2812 RGB LED strip with 4 LEDS
+// on an Digispark board.
+//
+// Replace neo in the code below to match the pin
+// that you are using if it is different.
+package main
+
+import (
+	"image/color"
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers/ws2812"
+)
+
+var leds [4]color.RGBA
+
+func main() {
+	led := machine.LED
+	led.Configure(machine.PinConfig{Mode: machine.PinOutput})
+
+	var neo machine.Pin = 0
```

Comment:
Perhaps the two ws2812 examples could be merged into one? You could move the LED array length and the NeoPixel pin number into separate arduino.go and digispark.go files in the examples/ws2812 package (with appropriate build tags).
Additionally, this definitely should have a smoke test.

Reply:
The part that seems to be important for the digispark variation is that `var leds [4]color.RGBA` is not inside of the `main()`.

Actually I can change the length back to 10 and it works fine.

Perhaps this can provide some clue about the nature of the backend problem.

In any case, I was not successful at getting the same example program to compile on both Arduino and Digispark.

Reply:
(not relevant anymore, but perhaps useful information)
You could define the `leds` array inside the main function while keeping the length outside:

```go
const numLeds = 4
func main() {
    var leds [numLeds]color.RGBA
    // ...
}
```

(The `const` is important here).